source/Mortgage.cpp

```diff
@@ -180,3 +180,17 @@ int64_t Mortgage::Payment() const
 	double power = pow(1. + interest, term);
 	return max<int64_t>(1, lround(principal * interest * power / (power - 1.)));
 }
+
+
+
+// Check the amount of the next payment due.
+double Mortgage::PrecisePayment() const
+{
+	if(!term)
+		return principal;
+	if(!interest)
+		return static_cast<double>(principal) / term;
+
+	double power = pow(1. + interest, term);
```

Comment:
```suggestion
	const double power = pow(1. + interest, term);
```
gotta love those const